dpnp/dpnp_iface_arraycreation.py

```diff
@@ -574,10 +807,23 @@ def copy(a, order="K", subok=False):
             f"default value ``False``, but got {subok}"
         )
 
-    if dpnp.is_supported_array_type(a):
+    if (
+        device is None
```

Comment:
We can use `dpnp_container.copy` not only with default values of CFD keywords but also when they match properties of input array `a` (like `a.usm_type == usm_type and ...`):
```python
if dpnp.is_supported_array_type(a):
    sycl_queue_normalized = dpnp.get_normalized_queue_device(a, device=device, sycl_queue=sycl_queue)
    if (usm_type is None or usm_type == a.usm_type) and sycl_queue_normalized == a.sycl_queue:
        return dpnp_container.copy(a, order=order)
```
Need to add examples with CFD keywords.
Also, interesting use case when `device == a.device and sycl_queue == None` but `sycl_queue_normalized != a.sycl_queue`.